File: BASIC/1052.py
# author -  newguo@sonaspy.cn
# coding - utf_8
import re
hand = re.split('[ \[\]]', input())
eye = re.split('[ \[\]]', input())
mouth = re.split('[ \[\]]', input())
n = hand.count('')
for i in range(n):
    hand.remove('')
n = eye.count('')
for i in range(n):
    eye.remove('')
n = mouth.count('')
for i in range(n):
    mouth.remove('')
mouth.remove('...')
num = eval(input())
for i in range(num):
    a = [int(j) - 1 for j in input().split()]
    s = ''
    flag = True
    if a[0] > len(hand)-1:
        print('Are you kidding me? @\/@')
        flag = False
    elif a[1] > len(eye)-1:
        print('Are you kidding me? @\/@')
        flag = False
    elif a[2] > len(mouth)-1:
        print('Are you kidding me? @\/@')
        flag = False
    elif a[3] > len(eye)-1:
        flag = False
        print('Are you kidding me? @\/@')
    elif a[4] > len(hand)-1:
        flag = False
        print('Are you kidding me? @\/@')
    if flag == True:
        s = hand[a[0]] +'('
        s = s + eye[a[1]]
        s = s + mouth[a[2]]
        s = s + eye[a[3]] + ')'
        s = s + hand[a[4]]
        print(s)

    

File: BASIC/1052.cpp
// author - sonaspy@outlook.com
// coding - utf_8

#include <iostream>
#include <string>
#define test() freopen("in", "r", stdin)

using namespace std;

int convert(string s, string a[])
{
    int k = 0;
    for (auto i : s)
        {
            if (i != '[' && i != ' ' && i != ']')
                a[k] += i;
            if (i == ']')
                k++;
        }
    return k;
}

int main(int argc, char const *argv[])
{
    /* code */
    //test();
    string s[3], hand[11], eye[11], mouth[11];
    getline(cin, s[0]);
    getline(cin, s[1]);
    getline(cin, s[2]);
    int l1 = convert(s[0], hand);
    int l2 = convert(s[1], eye);
    int l3 = convert(s[2], mouth);
    int N;
    cin >> N;
    int num[N][5];
    for (int i = 0; i < N; i++)
        for (int j = 0; j < 5; j++)
            cin >> num[i][j];
    for (int i = 0; i < N; i++)
        {
            int a1 = num[i][0], a2 = num[i][1], a3 = num[i][2], a4 = num[i][3], a5 = num[i][4];
            if (a1 <= l1 && a2 <= l2 && a3 <= l3 && a4 <= l2 && a5 <= l1 && a1 > 0 && a2 > 0 && a3 > 0 && a4 > 0 && a5 > 0)
                cout << hand[a1 - 1] << "(" << eye[a2 - 1] << mouth[a3 - 1] << eye[a4 - 1] << ")" << hand[a5 - 1] << endl;
            else
                cout << "Are you kidding me? @\\/@" << endl;
        }
    return 0;
}